src/plugins/intel_gna/tests/unit/transformations/gna_split_convolution_with_large_buffer_size.cpp

```diff
@@ -126,43 +131,41 @@ ngraph::Output<ngraph::Node> CreateConvolution::createOutputNode(const ngraph::O
 }
 
 // should be used only after CreateBaseDecorator
+template <const ngraph::Shape& kernel_shape, const ngraph::Shape& split_shape>
 class CreateSplittedConvolution : public CreateGraphDecorator {
 public:
-    CreateSplittedConvolution(CreateGraphDecoratorPtr prev,
-                              const ngraph::Shape& kernel_shape = ngraph::Shape{1, 64, 1, 1},
-                              const ngraph::Shape& split_shape = ngraph::Shape{960, 960, 960, 960, 256})
+    CreateSplittedConvolution(CreateGraphDecoratorPtr prev)
         : CreateGraphDecorator(std::move(prev)),
           kernel_shape_(kernel_shape),
           split_shape_(split_shape) {}
 
 protected:
-    void updateGraph(Graph& graph) override;
+    void updateGraph(Graph& graph) override {
```

Comment:
was it intentional?

Reply:
Providing method implementation in the class definition should be no problem for a .cpp file. As you can see, there is no common approach in this file regarding whether the method implementation should be inside the class definition or outside.

Reply:
but in my opinion it is harder to read what are properties of class.